core/src/actors/config_manager.rs

```diff
@@ -0,0 +1,82 @@
+use actix::{Actor, Context, Handler, Message, Supervised, SystemService};
+use log::{debug, info};
+use std::result;
+use witnet_config::loaders::toml;
+use witnet_config::Config;
+
+/// Config manager actor: manages the application configuration
+///
+/// This actor is in charge of reading the configuration for the
+/// application from a given source and using a given format, and
+/// supports messages for giving access to the configuration it holds.
+#[derive(Default, Debug)]
+pub struct ConfigManager {
```

Comment:
Please start this actor together with the rest of them [here](https://github.com/witnet/witnet-rust/blob/master/core/src/actors/node.rs)

Reply:
Currently, this actor requires that a `LoadConfig` message is sent to it to load a configuration. Do you think it's reasonable for this actor to have a `new(filename: String)` method so that the configuration can be loaded before the actor is even started? This way, there would be no need to send an initial message to the actor for it to load its configuration.

Reply:
As discussed, we know that in the near future we will have to rethink how to start the actors so that, if a failure occurs, they could recover gracefully. However, I would suggest to move (having the Config Manager using a `witnet.toml` file) in order to boost the integration of all the current Managers. :tada: What do you think guys?